frontend/src/app/patients/personSchema.ts

```diff
@@ -146,12 +146,12 @@ export function isValidBirthdate18n(t: TFunction) {
     }
     if (parsedDate.year() < 1900) {
       return this.createError({
-        message: t("patient.form.errors.birthDate.past"),
```

Comment:
By default this would return null, but this object will not allow nulls.

Reply:
thanks for fixing this!